INFO - 2023-12-28 23:19:00 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:19:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:19:05 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:19:05 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:19:23 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:19:23 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:19:23 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:19:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:19:31 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:19:31 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:19:31 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:19:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:19:43 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:19:43 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:19:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:19:45 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:19:45 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:19:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:19:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:19:54 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:19:54 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:20:04 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:20:10 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:20:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:20:21 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:20:21 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:20:27 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:20:27 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:20:27 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` IS NULL', [], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:20:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:20:46 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:20:46 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` IS NULL', [], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:20:54 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:20:59 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:20:59 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:20:59 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:21:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:21:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:21:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:21:22 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:21:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:21:38 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:21:38 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:22:18 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:22:18 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:22:18 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:22:21 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:22:25 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:22:25 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:22:25 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:22:42 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:22:42 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:22:42 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:22:43 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:22:45 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:22:46 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:22:50 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:22:50 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:22:50 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:22:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:22:52 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:23:11 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:23:15 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
ERROR - 2023-12-28 23:23:15 --> Error connecting to the database: mysqli_sql_exception: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:185
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(185): mysqli->real_connect('localhost', '', '', '', 3306, '', 0)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#4 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#7 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#9 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: Access denied for user ''@'localhost' (using password: YES) in C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:229
Stack trace:
#0 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `usernam...', Array, false)
#3 C:\wamp643.3.0\www\evaluation_system\app\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
#4 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(919): App\Controllers\Login->save()
#5 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#6 C:\wamp643.3.0\www\evaluation_system\vendor\codeigniter4\framework\system\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\wamp643.3.0\www\evaluation_system\public\index.php(79): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2023-12-28 23:23:15 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: YES)
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(574): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1615): CodeIgniter\Database\BaseConnection->query('SELECT `username`, `password`, `status`, `userID`, `fname`, `mname`, `lname`
FROM `users`
WHERE `username` = :username:', [...], false)
 3 APPPATH\Controllers\Login.php(35): CodeIgniter\Database\BaseBuilder->get()
 4 SYSTEMPATH\CodeIgniter.php(919): App\Controllers\Login->save()
 5 SYSTEMPATH\CodeIgniter.php(494): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 6 SYSTEMPATH\CodeIgniter.php(353): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(79): CodeIgniter\CodeIgniter->run()
INFO - 2023-12-28 23:24:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:24:05 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:25:37 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:25:44 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:25:47 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:25:51 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
INFO - 2023-12-28 23:25:53 --> Session: Class initialized using 'CodeIgniter\Session\Handlers\FileHandler' driver.
